2020-pip/RFP.md

```diff
@@ -0,0 +1,273 @@
+
+This project aims to complete the design, implementation, and rollout of pip's next-generation dependency resolver. This will lower the barriers to installing Python software, empowering users to get a version of a package that works. It will also lower the barriers to​ distributing ​Python software, empowering developers to make their work available in an easily reusable form.
+
+We plan to begin the project in Quarter 1 of 2020. Because of the size of the project, funding has been allocated to secure *two contractors*, a senior developer and an intermediate developer, to work on development, testing and building test infrastructure, code review, bug triage, and assisting in the rollout of necessary features.
+
+Please read this RFP and respond to let us know if you have questions, or submit a proposal if you are interested in performing the work for one of these contracts.
+
+## Timeline
+
+|Date|Milestone|
+|---|---|
+|November 11|Request for Proposal period opens.|
+|November 22|Request for Proposal period closes.|
+|November 27|Date proposals will have received a decision.|
+|December 31|Latest date for contracts for accepted proposals to be finalized.|
+|January 2|Latest date for contract work to commence.|
+
+
+## What is the Request for Proposals period?
+
+**A Request for Proposals (RFP)** is a process intended to allow us (the Python Software Foundation) to collect proposals from potential contractors and select contractor(s) best suited to fulfill the specified work.
+
+After the RFP period closes we will evaluate the received proposals based on the [evaluation criteria](#evaluation-criteria), seek clarification from proposers as necessary, and select one or more contractors to complete the work specified in the [scope section](#scope).
+
+**Note**: This Request For Proposal document may be updated to reflect things that we learn during the process. The canonical version and history is available [here](https://github.com/python/request-for/blob/master/2020-pip/RFP.md).
+
+## How do I submit a proposal?
+
+Proposals should be submitted as [Portable Document Format (PDF)](https://en.wikipedia.org/wiki/PDF) files via email to [ewdurbin@pyfound.org](mailto:ewdurbin@pyfound.org).
```

Comment:
Please include a Subject line prefix for identifying proposals.

Reply:
Thanks. Added.